sdks/python/apache_beam/runners/portability/portable_runner_test.py

```diff
@@ -284,6 +284,7 @@ def create_options(self):
     return options
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
```

Comment:
Can we add a TODO(BEAM-....) and include an error that happens in the Jira? Does the error happen on all Python versions? 
Similarly for other errors.